bin/git-theta-filter

```diff
@@ -60,26 +53,43 @@ def clean(args):
     }
     """
     logging.debug(f"Running clean filter on {args.file}")
-
     repo = git_utils.get_git_repo()
-    model_path = git_utils.get_relative_path_from_root(repo, args.file)
-    theta_model_dir = git_utils.get_relative_path_from_root(repo, git_utils.get_git_theta_model_dir(repo, model_path))
 
     checkpoint_handler = checkpoints.get_checkpoint_handler()
     model_checkpoint = checkpoint_handler.from_file(sys.stdin.buffer)
 
-    # TODO(bdlester): If we use Python3.7 as the minimum version, we don't need
-    # to use an OrderedDict as the standard dict retains the insertion order.
-    staged_file_contents = OrderedDict({"model_dir": theta_model_dir})
-    # Sort the keys so we don't get changing diffs based on serialization order.
-    for keys, param in sorted(utils.flatten(model_checkpoint).items()):
-        param_name = "/".join(keys)
-        tensor_metadata = OrderedDict({"shape": params.get_shape_str(param)})
-        tensor_metadata["dtype"] = params.get_dtype_str(param)
-        tensor_metadata["hash"] = params.get_hash(param)
-        staged_file_contents[param_name] = OrderedDict({"tensor_metadata": tensor_metadata})
+    obj = git_utils.get_file_version(repo, args.file, "HEAD")
+    if obj:
+        curr_metadata = metadata.Metadata.from_file(obj.data_stream).flatten()
```

Comment:
`prev_metadata` is probably a clearer name?

Reply:
Fixed